src/sentry/integrations/pipeline.py

```diff
@@ -334,6 +337,19 @@ def _dialog_response(self, data, success) -> HttpResponseBase:
         )
         return render_to_response("sentry/integrations/dialog-complete.html", context, self.request)
 
+    def _enable_actions(self) -> None:
+        """
+        Enables all disabled actions for the integration.
```

Comment:
```suggestion
        Enables all actions for the integration.
```

Since this isn't actually filtering to disabled actions, is it possible there could be a case where the action is pending deletion or something and we don't want to take it out of that state?